podvm/README.md

```diff
@@ -73,48 +74,82 @@ $ docker build -t podvm_builder \
 	-f Dockerfile.podvm_builder .
 ```
 
-## Building the podvm qcow2 image
+## Building the image containing the podvm binaries
 
-The process to build the podvm container image will effectively compile binaries and create the qcow2 file.
+In order to build the binaries you should be using the corresponding dockerfile
+of the Linux distro for which the builder image was built.
+For example, if the builder image was built with *Dockerfile.podvm_builder.DISTRO* then
+you should use the *Dockerfile.podvm_binaries.DISTRO* to build the image with podvm binaries.
 
-In order to build the podvm image you should be using the corresponding dockerfile of the Linux distro for which the builder image was built. For example, if the builder image was built with *Dockerfile.podvm_builder.DISTRO* then you should use the *Dockerfile.podvm.DISTRO* to build the podvm image.
+Assuming you have built the podvm_builder image for Ubuntu as explained in the previous step,
+running the following command will build the image with the podvm binaries.
 
-The builder image has to be indicated via `BUILDER_IMG` build argument to docker. Apart from that, the `CLOUD_PROVIDER` argument is mandatory and specifies the cloud provider that binaries should be built for. Once again, one builder image can be used to build for multiple
-cloud providers and architectures.
+```
+$ docker build -t podvm_binaries \
+         --build-arg BUILDER_IMG=podvm_builder \
+	 -f Dockerfile.podvm_binaries .
 
-Below is shown how to build an Ubuntu image for libvirt:
+```
+The build process can take significant time.
```

Comment:
Sorry to nitpick, but I think this build is sensitive to architecture too? If so it might be worth adding a sentence about the option to setting `--build-arg ARCH`?